.github/workflows/release-pr.yml

```diff
@@ -92,8 +92,8 @@ jobs:
           BRANCH_NAME: ${{ steps.create_branch.outputs.branch-name }}
         run: |
           npm install # To update package-lock.json with the new version of the plugin
-          git config user.name "${{ github.actor }}"
-          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Would it be safer to somehow fetch the id in the email from [here](https://api.github.com/users/github-actions%5Bbot%5D) and then use that in case it changes? 🤔 Shouldn't happen ideally but since this is undocumented behavior by GH, I am not too sure about this.

Reply:
Worst case scenario, this ID is not valid anymore and we might lose the nice GitHub logo next to the author in the commits.
I don't think it's worth going into this trouble for so little reward (and I don't think this ID will change anytime soon)

But ideally, yes, we'd do that if we want to be future-proof

Reply:
Something to keep in mind in case it starts failing, resolving this for now.

Reply:
Agreed, left a comment above